package.json

```diff
@@ -11,32 +13,29 @@
   "license": "MIT",
   "private": false,
   "scripts": {
-    "cover": "mastarm test --coverage --coverage-paths index.js",
-    "generate-docs": "documentation readme index.js --document-exported=true -g --section=API",
-    "flow": "mastarm flow",
-    "lint": "mastarm lint",
-    "lint-docs": "documentation lint index.js",
-    "prepublish": "mastarm prepublish index.js:build/index.js",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
-    "test": "yarn run lint && yarn run flow && yarn run lint-docs && mastarm test",
-    "test-node": "mastarm test --test-environment node"
-  },
-  "mastarm lint": {
-    "ignore": [
-      "flow-typed"
-    ]
+    "build": "tsdx build -i index.ts --target node",
+    "cover": "tsdx test --coverage --coverage-paths index.ts",
+    "generate-docs": "documentation readme index.ts --document-exported=true -g --section=API",
+    "jest": "yarn build && tsdx test",
+    "lint": "tsdx lint __tests__ index.ts --quiet",
+    "lint-docs": "documentation lint index.ts",
+    "prepublish": "yarn build",
+    "semantic-release": "semantic-release",
+    "test": "yarn run lint && yarn run lint-docs && yarn jest",
+    "test-node": "yarn build && tsdx test --test-environment node"
```

Comment:
Is this still needed?

Reply:
[It's used in the CI](https://github.com/miles-grant-ibigroup/geocoder-arcgis-geojson/blob/c0b51cdc2ccce186ed6109d089331898c18e81d6/.github/workflows/node-ci.yml#L27). Not sure if `yarn test` would work just as well, but didn't want to create more changes than needed. You think it's worth changing?

Reply:
I see how it works now in CI, we can keep as is.